src/Form/FullFileDeletionForm.php

```diff
@@ -57,14 +62,55 @@ abstract class FullFileDeletionForm extends ContentEntityConfirmFormBase {
    */
   protected $mediaStorage;
 
+  /**
+   * The purger factory service.
+   *
+   * @var \Drupal\purge\Plugin\Purge\Invalidation\InvalidationsService
```

Comment:
Hey @anthony-malkoun @vincent-gao,

This change introduces a dependency on the tide_site and purge modules in order to be able to lookup and invalidate URLs - I'm not sure how to handle this. These modules aren't included in this project's composer.json so requiring them in info.yml will cause local builds and CI environments to fail.

Reply:
Hey @FleetAdmiralButter , I think we probably could send out an `Event` or a custom hook that contains `file ids` and `site_ids` etc... so that other modules will have an opportunity to alter or to deal with this date.